x-pack/plugins/maps/public/components/help_menu.js

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Not sure if this file should go here. `public/components` is for redux controlled components. Maybe this file should go in `public/shared/components`?

Reply:
I think `public/components` should be renamed to `public/connected_components` to make that clear but that is for another issue